src/analysis/if_statements.d

```diff
@@ -117,6 +117,17 @@ private struct ExpressionInfo
 			return 0;
 		return -1;
 	}
+	bool opEquals(ref const ExpressionInfo other) const nothrow
+	{
+		return formatted == other.formatted
+			&& line == other.line
+			&& column == other.column
+			&& depth == other.depth;
```

Comment:
Doesn't this contradict the hash function, because it takes the `formatted` into account?